src/App.tsx

```diff
@@ -1,26 +1,251 @@
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">
-          React Todo App - Load Todos
-        </a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <>
+      <div className="todoapp">
+        <h1 className="todoapp__title">todos</h1>
+        <div className="todoapp__content">
+          <header className="todoapp__header">
+            <button
+              type="button"
+              className={`todoapp__toggle-all ${allCompleted ? 'active' : ''}`}
+              data-cy="ToggleAllButton"
+              onClick={completedAllTodos}
+            />
+
+            <form onSubmit={handleSubmit}>
+              <input
+                value={newTodoTitle}
+                ref={inputRef}
+                onChange={e => setNewTodoTitle(e.target.value)}
+                data-cy="NewTodoField"
+                type="text"
+                className={`todoapp__new-todo ${inputRef.current?.focus() ? 'focused' : ''}`}
+                placeholder="What needs to be done?"
+                disabled={tempTodo !== null}
+                autoFocus
+              />
+            </form>
+          </header>
+        </div>
+        <Todolist
+          todos={filteredTodos}
+          onToggleTodo={toggleTodoCompletion}
+          onDelete={handleDelete}
+          deletingId={deletingId}
+          loading={loading}
+        />
+        {tempTodo && (
+          <TodoItem
+            todo={tempTodo}
+            onToggleTodo={toggleTodoCompletion}
+            onDelete={handleDelete}
+            deletingId={deletingId}
+            loading={loading}
+          />
+        )}
+        {todos.length !== 0 && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {`${activeTodos} items left`}
+            </span>
+
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={`filter__link ${currentFilter === FilterOptions.All ? 'selected' : ''}`}
+                data-cy="FilterLinkAll"
+                onClick={() => setCurrentFilter(FilterOptions.All)}
+              >
+                All
+              </a>
+              <a
+                href="#/active"
+                className={`filter__link ${currentFilter === FilterOptions.Active ? 'selected' : ''}`}
+                data-cy="FilterLinkActive"
+                onClick={() => setCurrentFilter(FilterOptions.Active)}
+              >
+                Active
+              </a>
+              <a
+                href="#/completed"
+                className={`filter__link ${currentFilter === FilterOptions.Completed ? 'selected' : ''}`}
+                data-cy="FilterLinkCompleted"
+                onClick={() => setCurrentFilter(FilterOptions.Completed)}
+              >
+                Completed
+              </a>
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method